pkg/podsecurityadmission/podsecurityadmission_test.go

```diff
@@ -48,19 +48,33 @@ func TestGetAdmissionConfigFromCluster(t *testing.T) {
 
 func TestGetPluginConfigFromTemplate(t *testing.T) {
 	tests := []struct {
-		testName string
-		source   *v3.PodSecurityAdmissionConfigurationTemplate
-		expected v1.AdmissionPluginConfiguration
+		testName       string
```

Comment:
Add some test cases for invalid versions.

Reply:
The test case is added.